components/Navbar.tsx

```diff
@@ -101,6 +101,12 @@ const routes: NavSection[] = [
       {
         name: "Compute Infrastructure",
```

Comment:
The two services columns (Compute Infrastructure and Research Support & Consulting) need to switch sides also.